src/melee/gr/stage.h

```diff
@@ -119,7 +113,7 @@ typedef struct _StageInfo {
             GXColor color;
         }* ptr;
     }* x12C;
-    Vec3 x130, x13C, x148, x154, x160, x16C;
+    Vec x130, x13C, x148, x154, x160, x16C;
```

Comment:
Is `Vec3` gone everywhere now?

Reply:
No, only in this header - just wanted to get rid of that weird #ifndef M2CTX